ButtonMerchant.xcodeproj/project.pbxproj

```diff
@@ -1535,7 +1535,7 @@
 				SKIP_INSTALL = YES;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
 				SWIFT_TREAT_WARNINGS_AS_ERRORS = YES;
-				SWIFT_VERSION = 4.0;
+				SWIFT_VERSION = 5.0;
```

Comment:
Do all targets need to be updated to swift 5 as well?

Noticing some targets had warnings

## Example

```
UIApplicationLaunchOptionsKey -> UIApplication.LaunchOptionsKey
UIApplicationOpenURLOptionsKey -> UIApplication.OpenURLOptionsKey

// Function parameter type updated: Any -> UIUserActivityRestoring
func application(_ application: UIApplication, continue userActivity: NSUserActivity, restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
        ButtonMerchant.trackIncomingUserActivity(userActivity)
        return true
    }
```

## Example-objc
```
- (BOOL)application:(UIApplication *)application continueUserActivity:(NSUserActivity *)userActivity restorationHandler:(void(^)(NSArray<id<UIUserActivityRestoring>> * __nullable restorableObjects))restorationHandler {
    [ButtonMerchant trackIncomingUserActivity:userActivity];

    return YES;
}
```

## IntegrationTests
```
let object = responseDict["object"] as? [String: Any],
```